AnnoyancesFilter/Popups/sections/antiadblock.txt

```diff
@@ -532,6 +532,7 @@ w3resource.com#@#.ad-placement
 ! SECTION: Anti-adblock - Regular rules
 ! Do not forget about '!#safari_cb_affinity' section for exclusions
 !
+fosslinux.com#%#//scriptlet('set-constant', 'penci_adlbock', 'undefined')
```

Comment:
Your scriptlet is fine, but what about content blockers?
I'd say these rules are enough. Add the exceptions to `! SECTION: Anti-adblock - The rules for content blockers only`.
```adblock
fosslinux.com#@#.adsbygoogle
fosslinux.com#@#.sidebar-ad
fosslinux.com#@#.Ad-Container
fosslinux.com#$#.adsbygoogle.Ad-Container.sidebar-ad { display: block !important; }
```

Reply:
By the way, Is not CSS rule not supported by AdGuard Content Blocker?

Reply:
> By the way, Is CSS rule not supported by AdGuard Content Blocker?

As I recall, only the basic rules like `##` or network ones`||`.